fission-server/src/models.rs

```diff
@@ -0,0 +1,32 @@
+//! Structs
+
+#![allow(missing_docs)]
```

Comment:
this necessary?

Reply:
I didn't want to describe every single field of these. Not sure? 🤷‍♂️

Reply:
@icidasset ah. copilot can be handy for these things tbh haha.

Reply:
I think you can do it at the struct level, but no biggie.